contrib/kubespray/preinstall.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+while getopts "c:" opt; do
+  case $opt in
+    c)
+      CLUSTER_CONFIG=$OPTARG
+      ;;
+    \?)
```

Comment:
```suggestion
    *)
```

Reply:
According to https://sookocheff.com/post/bash/parsing-bash-script-arguments-with-shopts/, it works